AllHitAnalyze_New.py

```
@@ -1,162 +1,162 @@
+				# Great, (-150, -90] & [+90, +150) ms
+				elif hitFAbs < 150:
+					self.Accurate_Sync_Rate[5] += 1;
+				# Right, [+150, +250) ms
+				elif hitFAbs < 250:
+					self.Accurate_Sync_Rate[6] += 1;
+				# Miss, [+250, +∞) ms
+				else:
+					self.Accurate_Sync_Rate[7] += 1;
+				# add yAxis list
+				if hitI < 250:
+					self.yAxis[150 + hitI] += 1;
```

Ruff (0.14.8)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)